src/core/ticker/Ticker.js

```diff
@@ -321,7 +374,30 @@ export default class Ticker
             this.deltaTime = elapsedMS * settings.TARGET_FPMS * this.speed;
 
             // Invoke listeners added to internal emitter
-            this._emitter.emit(TICK, this.deltaTime);
+            let listener = this._head.next;
+
+            while (listener)
+            {
+                const previous = listener.previous;
+
+                listener.emit(this.deltaTime);
+
+                // Once listener or destroyed itself
+                if (listener.destroyed)
+                {
+                    listener = previous.next;
+                }
+                else
+                {
+                    // Go to the next listener
+                    listener = listener.next;
+                }
+            }
```

Comment:
Although I'm ok with basically all of this, I would highly recommend a benchmark, as property lookups are slower than you think, which in JavaScript, actually makes iterating a large linked list a lot slower than a large array (there is already a jsperf for this, I'm just too lazy to find it at the moment, sorry). In the grand scheme of things though, especially this use case since this is kind of a stop-gap in a lot of people's minds anyway, it's probably fine, but just not in the vein of a mini-runner kind of performance mindedness.

So all that being said, I still think you can squeeze a little perf out of this while loop right now, if you just do this in the `TickerListener#destroy` method:
```js
destroy()
{
    if (this.destroyed)
    {
        return;
    }
    this.destroyed = true;
    this.fn = null;
    this.context = null;
    this.disconnect();
    // Optimize emit loop, this is safe
    this.next = this.previous;
}
```
You do not need to worry about `this` holding on to `previous` in `next` property, because nothing holds a reference to `this` anymore, when the `emit` method unwinds, `this` can and will be marked for GC regardless of holding a reference in `next`. So this can just optimize for the emit loop.

So your loop can look like this:
```js
while (listener)
{
    listener.emit(this.deltaTime);
    listener = listener.next;
}
```
If you're feeling saucy 🤡, `this.next` can be returned from `TickerListener#emit`:
```js
while (listener)
{
    listener = listener.emit(this.deltaTime);
}
```